src/pfe/file-watcher/server/src/controllers/projectEventsController.ts

```diff
@@ -42,6 +42,30 @@ const workspaceSettingsInfo =  workspaceSettings.workspaceSettingsInfoCache ? JS
 // default 20s timeout to wait for all chunks
 const timeout = (workspaceSettingsInfo && workspaceSettingsInfo.watcherChunkTimeout) ? workspaceSettingsInfo.watcherChunkTimeout : 20000;
 
+/**
+ * Determine if Codewind should handle a detected file change, or if it is handled by an extension
+ *
+ * @param detectChangeByExtension property from the project handler
+ * @param path the path of detected file change
+ */
+function shouldHandle(detectChangeByExtension: boolean | string[], path: string): boolean {
+
+    // clearly Codewind should handle when detectChangeByExtension is false
+    if (!detectChangeByExtension) {
```

Comment:
Will the code be simpler if you just check if it is either true or false, then just return at this point. If it falls through, then it is an array.

Reply:
Thanks for the suggestion. Updated the logic.